README.md

```diff
@@ -46,6 +48,13 @@ To enable plans using this module, follow these steps based on the subscription
 * To disable all plans execute `terraform destroy` command.
 * To disable a specific plan, remove the plan name from mdc_plans_list var and execute `terraform apply` command.
 
+### Already onboarded subscription
```

Comment:
Maybe we can add an explanation that we recommend cleaning the subscription and manage all the onboarding process with our module and then give the options. 

Another suggestion, we could split the explanation into two parts:
If you already onboarded using the UI -> give option 4
If you already onboarded using Terraform -> option 1,2,3